src/dmd/backend/elfobj.d

```diff
@@ -121,6 +121,8 @@ enum ELF_COMDAT = TARGET_LINUX;
  *      R_386_PC32      R_X86_64_PC32     R_X86_64_
  */
 
+alias reltype_t = uint;
```

Comment:
Shouldn't https://github.com/dlang/dmd/blob/eda232a9f72b482c92cefb27a4321afc0aedbb5c/src/dmd/backend/melf.d#L409-L437 be made an `enum` instead ?

Reply:
I made enough changes for one PR. There's always more.

Reply:
You could always submit a separate PR which makes the aforementioned code an enum, then rebase this one on top once its merged.